### PR TITLE
Feat: [IGL-95] Filter rebrand

### PR DESCRIPTION
**Pull request by vicky-comeau**

### CODE REVIEW

File: .storybook/useBrand.js [hunk 1]
```diff
@@ -2,62 +2,10 @@ import { useEffect } from 'react';
 
 export const useBrand = (StoryFn, context) => {
     const {brand} = context.globals;
-    const {displayName} = context.component;
-    const components = [
```
Comment (fraincs) [inline .storybook/useBrand.js:6]: I'm out of the loop here, what's the goal of this removal?

File: packages/Color/src/Color.tsx [hunk 2]
```diff
@@ -1,81 +1,99 @@
...
-          break;
+    const classes = cx("ids-color", `ids-color--${size}`, className);
+
+    const initialsFromName = React.useMemo(() => {
+        if (!name) {
+            return null;
+        }
+
+        let initials = "";
+        const splitName = name.trim().split(" ");
```
Comment (fraincs) [inline packages/Color/src/Color.tsx:48]: If the name has more than one space this won't work.

File: packages/ColorPicker/src/ColorPicker.tsx [hunk 3]
```diff
@@ -14,26 +14,160 @@ import ColorPickerOption from "./ColorPickerOption";
...
+    { 
+        id: "electricBlue700", 
+        value: variables.electricBlue700, 
+        hoverColor: variables.electricBlue700, 
+        iconColor: variables.samoyed 
+    }
 ] as const;
 
-export type ColorName = typeof colorNames[number]["id"];
+export const colorNamesWL = [
```
Comment (fraincs) [inline packages/ColorPicker/src/ColorPicker.tsx:111]: I would use Workleap here as we are not using a shorter name for Igloo.

File: packages/ColorPicker/src/ColorPicker.tsx [hunk 4]
```diff
@@ -47,15 +181,36 @@ export interface ColorPickerProps {
...
+    const brand = propBrand ?? document.documentElement.getAttribute("data-brand") ?? "igloo";
+    
+    const getDefaultColor = (): ColorName => {
+        return brand === "workleap" ? "decorativeOption3" : "dandelion200";
+    };
+
+    const selectedColor = propSelectedColor ?? getDefaultColor();
+
+    let colorNames: ReadonlyArray<Readonly<ColorNameItem>>;
+    if (brand === "workleap") {
```
Comment (fraincs) [inline packages/ColorPicker/src/ColorPicker.tsx:200]: I would add a return before the if

File: packages/ColorPicker/src/__snapshots__/ColorPicker.test.tsx.snap [hunk 5]
```diff
@@ -9,11 +9,13 @@ exports[`ColorPicker It should render a snapshot 1`] = `
     <button
       class="ids-color-picker__button"
       data-color="dandelion200"
+      data-rac=""
```
Comment (fraincs) [inline packages/ColorPicker/src/__snapshots__/ColorPicker.test.tsx.snap:12]: What is it used for?

File: packages/ColorPicker/src/color-picker-option.scss [hunk 6]
```diff
@@ -15,9 +15,23 @@
...
-  
-  /* Icon */
-  --ids-color-picker-option-icon-color: #{tokens.$samoyed};
+
+}
+
+[data-brand="workleap"] {
+  /* Default */
+  --ids-color-picker-option-border-radius: var(--hop-shape-circle);
+  --ids-color-picker-option-border-size: 0.09rem;
```
Comment (fraincs) [inline packages/ColorPicker/src/color-picker-option.scss:24]: 1.44px seems arbitrary.

File: packages/ColorPicker/src/color-picker.scss [hunk 7]
```diff
@@ -12,13 +15,33 @@
...
   --ids-color-picker-btn-border-size: 0.15rem;
-  --ids-color-picker-btn-focus: #{tokens.$focus};
+  --ids-color-picker-btn-focus: #{tokens.$electric-blue-600};
   --ids-color-picker-btn-padding: #{tokens.$space-2};
 }
 
+[data-brand="workleap"] {
+  /* Default */
+  --ids-color-picker-font-family: var(--hop-body-sm-font-family);
+  --ids-color-picker-font-size: var(--hop-body-sm-font-size);
```
Comment (fraincs) [inline packages/ColorPicker/src/color-picker.scss:29]: Font weight and line-height are missing.

File: .storybook/useBrand.js (hunk 1 above)
Comment (vicky-comeau) [reply .storybook/useBrand.js:6]: We did the rebrand everywhere, so we don't need to specify specific components anymore.

File: packages/ColorPicker/src/ColorPicker.tsx (hunk 4 above)
Comment (vicky-comeau) [reply packages/ColorPicker/src/ColorPicker.tsx:200]: not sure I understand, a return for what?

File: packages/ColorPicker/src/color-picker-option.scss (hunk 6 above)
Comment (vicky-comeau) [reply packages/ColorPicker/src/color-picker-option.scss:24]: Well it called for 1.5px so I rounded 0.09375rem. That's just way too long. Plus browsers don't end up computing the exact amount anyway.

File: packages/ColorPicker/src/__snapshots__/ColorPicker.test.tsx.snap (hunk 5 above)
Comment (vicky-comeau) [reply packages/ColorPicker/src/__snapshots__/ColorPicker.test.tsx.snap:12]: This is a react-aria button so I'm guessing it's added by them. I double checked the code, not something we added.

Comment (alexasselin008) [reply packages/ColorPicker/src/__snapshots__/ColorPicker.test.tsx.snap:12]: react-aria-components (rac) seems to be adding this property on react aria components. 

There is a major release releasing todayt of react-aria-components, and the code that adds the data-rac is still there. I think it's used by their tailwindcss integration to target react-aria-components. 

https://github.com/adobe/react-spectrum/blob/9ce2f674eab2cc8912800d3162dcf00a1ce94274/packages/react-aria-components/src/utils.tsx#L134-L135

Comment (alexasselin008) [reply packages/ColorPicker/src/__snapshots__/ColorPicker.test.tsx.snap:12]: in all cases, don't worry about it for now i think

File: packages/ColorPicker/src/color-picker-option.scss (hunk 6 above)
Comment (fraincs) [reply packages/ColorPicker/src/color-picker-option.scss:24]: True in that case we might need to ask design to stop using .5px for borders 😄

File: packages/ColorPicker/src/ColorPicker.tsx (hunk 4 above)
Comment (fraincs) [reply packages/ColorPicker/src/ColorPicker.tsx:200]: new line

File: packages/ColorPicker/src/color-picker-option.scss (hunk 6 above)
Comment (vicky-comeau) [reply packages/ColorPicker/src/color-picker-option.scss:24]: haha maybe... I guess I'll keep it for now?

**Review by fraincs — APPROVED** [review]

**Review by franckgaudin — APPROVED** [review]

**Review by fraincs — APPROVED** [review]

**Pull request merged**